lib/History.js

```diff
@@ -10,13 +10,25 @@ class History extends Request {
 
   // GET /api/history/period/<timestamp>
   // https://home-assistant.io/developers/rest_api/#get-apihistoryperiodlttimestamp
-  state(timestamp, filter) {
-    if (moment(timestamp, 'YYYY-MM-DDTHH:mm:ssZ', true).isValid() === false) {
+  state(timestamp, filter, endTimestamp) {
+    if (timestamp !== undefined && moment(timestamp, 'YYYY-MM-DDTHH:mm:ssZ', true).isValid() === false) {
       return Promise.reject(new Error('Invalid timestamp format.'));
```

Comment:
Let's change all instances of `timestamp` to `startTimestamp` to avoid confusion. Update the error messages to simply state "Invalid startTimestamp format." (And endTimestamp for the error message on line 18.)

Reply:
Good point, changed.